service/src/main/java/tds/config/web/advice/ExceptionAdvice.java

```diff
@@ -0,0 +1,47 @@
+package tds.config.web.advice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import tds.config.web.resources.ExceptionMessageResource;
+
+import java.util.NoSuchElementException;
+
+@ControllerAdvice
+public class ExceptionAdvice {
+    private static final Logger LOG = LoggerFactory.getLogger(ExceptionAdvice.class);
+
+    // TODO:  Find better/more appropriate exception?  Make our own?
```

Comment:
I think we should roll our own NotFoundException.  Reading the JavaDoc around NoSuchElementException it would be confusing to me to see that being thrown in a query method in the service.